packages/terra-alert/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -0,0 +1,43 @@
+:global {
+
+  // ---------------------- Shared ----------------------- //
+
+  --terra-alert-alert-background-color: #332425;
+  --terra-alert-alert-title-color: #fb4c4c;
+
+  --terra-alert-custom-background-color: #1a2226;
+  --terra-alert-custom-text-color: #b2b5b6;
+
+  --terra-alert-error-background-color: #332425;
+  --terra-alert-error-title-color: #fb4c4c;
+
+  --terra-alert-advisory-background-color: #1a2226;
+  --terra-alert-info-background-color: #1a2226;
+  --terra-alert-success-background-color: #1a2226;
+  --terra-alert-warning-background-color: #464232;
+
+  // -------------------- Version 4.x -------------------- //
+  --terra-alert-custom-default-color: #1c1f21;
+
+  // -------------------- Version 3.x -------------------- //
+
+  --terra-alert-color: #b2b5b6;
+
+  --terra-alert-advisory-box-shadow: 2px 0 0 #5a81ff inset;
+  --terra-alert-alert-box-shadow: 2px 0 0 #fb4c4c inset;
+  --terra-alert-custom-box-shadow: 2px 0 0 #9b9fa1 inset;
```

Comment:
This box shadow color should not be defined for the custom box shadow. By doing this, it overrides the custom color defined via props: https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.module.scss#L127-L129

Reply:
After referring to the issue https://github.com/cerner/terra-core/issues/2788 and the associated PR. It seems that the variable `--terra-alert-custom-box-shadow` is being removed. So, i have removed this variable from lowlight theme as well as seen here : https://github.com/cerner/terra-core/pull/2758/commits/0c31e51cd29f1f795a5fe6b6832f6af8e4356048

Reply:
I have reverted the changes from https://github.com/cerner/terra-core/commit/0c31e51cd29f1f795a5fe6b6832f6af8e4356048 here : https://github.com/cerner/terra-core/pull/2758/commits/f0b4184ca3d941e38b02ceb6ee8fb5f386d9efd2.
Since this issue also persists in fusion theme and will be addressed as part of issue https://github.com/cerner/terra-core/issues/1882. So, in order to keep it consistent and simple, it makes more sense to address this issue in lowlight theme as part of https://github.com/cerner/terra-core/issues/1882

Reply:
@avinashg1994 good catch! thank you!!!